specification/botservice/resource-manager/Microsoft.BotService/preview/2021-05-01/botservice.json

```diff
@@ -0,0 +1,2976 @@
+          "type": "boolean"
+        },
+        "message": {
+          "description": "additional message from the bot management api showing why a bot name is not available",
+          "type": "string"
+        }
+      }
+    },
+    "HostSettingsResponse": {
+      "description": "The response body returned for a request to Bot Service Management to check per subscription hostSettings",
+      "type": "object",
+      "properties": {
+        "oAuthUrl": {
+          "description": "For in-conversation bot user authentication",
+          "type": "string"
+        },
+        "toBotFromChannelOpenIdMetadataUrl": {
+          "description": "For verifying incoming tokens from the channels",
+          "type": "string"
+        },
+        "toBotFromChannelTokenIssuer": {
+          "description": "For verifying incoming tokens from the channels",
+          "type": "string"
+        },
+        "toBotFromEmulatorOpenIdMetadataUrl": {
+          "description": "For verifying incoming tokens from bot emulator",
+          "type": "string"
+        },
+        "toChannelFromBotLoginUrl": {
+          "description": "For getting access token to channels from bot host",
+          "type": "string"
+        },
+        "toChannelFromBotOAuthScope": {
+          "description": "For getting access token to channels from bot host",
+          "type": "string"
+        },
+        "validateAuthority": {
+          "description": "Per cloud OAuth setting on whether authority is validated",
+          "type": "boolean"
+        },
+        "botOpenIdMetadata": {
+          "description": "Same as toBotFromChannelOpenIdMetadataUrl, used by SDK < v4.12",
+          "type": "string"
+        }
+      }
+    },
+    "OperationResultsDescription": {
+      "description": "The properties indicating the operation result of an operation on a service.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The ID of the operation returned.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "name": {
+          "description": "The name of the operation result.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "status": {
+          "description": "The status of the operation being performed.",
+          "enum": [
+            "Canceled",
+            "Succeeded",
+            "Failed",
+            "Requested",
+            "Running"
+          ],
+          "readOnly": true,
+          "type": "string",
+          "x-ms-enum": {
+            "name": "OperationResultStatus",
+            "modelAsString": true
+          }
+        },
+        "startTime": {
+          "description": "The time that the operation was started.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "properties": {
+          "x-ms-client-flatten": false,
+          "description": "Additional properties of the operation result."
```

Comment:
Is this leftover? You need to schematize whatever other properties you want. They shouldn't be in a nested property bag.

Reply:
In that case, please remove it. You'll need a new API version to add new properties anyway, so add whatever you need when you create your next API version.

Reply:
> In that case, please remove it. You'll need a new API version to add new properties anyway, so add whatever you need when you create your next API version.

You may have missed this one. Strongly recommend only defining objects that you need.